_resources/port1.0/group/python-1.0.tcl

```diff
@@ -295,12 +295,13 @@ proc python_set_pep517 {option action args} {
         return
     }
     global python.pep517 python.version subport name
-    if {$subport ne $name} {
+    if {[info exists python.version]} {
```

Comment:
what's the reason for this change?

Reply:
Without that change, you cannot use PEP-517 builds in single-version ports using `python.version` directly, such as `pipenv`.

Reply:
I'll look at refactoring this. It would probably work better as a callback anyway.